### PR TITLE
FF131ExperFeat: CSSPositionTryRule/TryDescriptors

### PR DESCRIPTION
**Pull request by hamishwillee**
FF131 implements [`CSSPositionTryRule`](https://developer.mozilla.org/en-US/docs/Web/API/CSSPositionTryRule) and [`CSSPositionTryDescriptors`](https://developer.mozilla.org/en-US/docs/Web/API/CSSPositionTryDescriptors) behind a preference `layout.css.anchor-positioning.enabled` in https://bugzilla.mozilla.org/show_bug.cgi?id=1909346

This a very small part of the CSS anchor positioning module docs. I've added a section to Experimental features so that we can grow this as the functionality is built out. BUT, I have not added to release notes as you really can't do anything useful with this yet. So the experimental features section is just so that we have a starting point for later.

I also updated the docs very slightly to match web api conventions.

Related docs work can be tracked in #35699

### CODE REVIEW

File: files/en-us/web/api/csspositiontrydescriptors/index.md [hunk 1]
```diff
@@ -9,66 +9,101 @@ browser-compat: api.CSSPositionTryDescriptors
...
+> [!NOTE]
+> The {{domxref("CSSPositionTryRule")}} interface represents a {{cssxref("@position-try")}} at-rule, and the {{domxref("CSSPositionTryRule.style")}} property is an instance of this object.
 
 {{InheritanceDiagram}}
 
 ## Instance properties
 
 _Inherits properties from its ancestor {{domxref("CSSStyleDeclaration")}}._
 
-The `CSSPositionTryDescriptors` interface defines properties that represent the CSS properties that can be set inside a {{cssxref("@position-try")}} at-rule.
```
Comment (hamishwillee) [inline files/en-us/web/api/csspositiontrydescriptors/index.md:20]: This was a pragmatic way to do the docs, but it is like nothing in web API. I have modeled this more on [CSSPageDescriptors](https://developer.mozilla.org/en-US/docs/Web/API/CSSPageDescriptors) (and every other API doc) and split this into a definition list. 

I moved the explanation of these into the description.

File: files/en-us/web/api/csspositiontryrule/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ _Inherits properties from its ancestor {{domxref("CSSRule")}}._
 - {{domxref("CSSPositionTryRule.name")}} {{ReadOnlyInline}} {{experimental_inline}}
   - : Represents the name of the position try option specified by the `@position-try` at-rule's {{cssxref("dashed-ident")}}.
 - {{domxref("CSSPositionTryRule.style")}} {{ReadOnlyInline}} {{experimental_inline}}
-  - : Returns a {{domxref("CSSPositionTryDescriptors")}} object representing the declarations set in the body of the `@position-try` at-rule.
+  - : A {{domxref("CSSPositionTryDescriptors")}} object that represents the declarations set in the body of the `@position-try` at-rule.
```
Comment (hamishwillee) [inline files/en-us/web/api/csspositiontryrule/index.md:23]: FYI, preference not to use `Returns`, even on read only properties (I don't care strongly, but  I'm fixing the cases I see).

File: files/en-us/web/api/csspositiontrydescriptors/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/api/csspositiontrydescriptors/index.md:20]: This looks way better, IMO

File: files/en-us/web/api/csspositiontrydescriptors/index.md [hunk 3]
```diff
@@ -9,66 +9,101 @@ browser-compat: api.CSSPositionTryDescriptors
...
-| {{cssxref("min-width")}}           | `CSSPositionTryDescriptors.minWidth`          | `CSSPositionTryDescriptors["min-width"]`           |
-| {{cssxref("place-self")}}          | `CSSPositionTryDescriptors.placeSelf`         | `CSSPositionTryDescriptors["place-self"]`          |
-| {{cssxref("position-anchor")}}     | `CSSPositionTryDescriptors.positionAnchor`    | `CSSPositionTryDescriptors["position-anchor"]`     |
-| {{cssxref("right")}}               | `CSSPositionTryDescriptors.right`             | `CSSPositionTryDescriptors["right"]`               |
-| {{cssxref("top")}}                 | `CSSPositionTryDescriptors.top`               | `CSSPositionTryDescriptors["top"]`                 |
-| {{cssxref("width")}}               | `CSSPositionTryDescriptors.width`             | `CSSPositionTryDescriptors["width"]`               |
+The property names in snake-case and camel-case are:
+
+- `align-self` or `alignSelf`
+  - : A string representing the value of an {{cssxref("align-self")}} descriptor in the corresponding `@position-try` at-rule.
```
Comment (bsmth) [inline files/en-us/web/api/csspositiontrydescriptors/index.md:30]: We have a lot of duplication of "in the corresponding `@position-try` at-rule.", should we move that before the DL?

Comment (hamishwillee) [reply files/en-us/web/api/csspositiontrydescriptors/index.md]: Good suggestion. Done.

**Review by bsmth — APPROVED** [review]
Thanks, Hamish, +1 from me!

**Pull request merged**